### DISCUSSION TITLE
Cannot resolve favicon.ico GET 500 for file-based metadata

### DISCUSSION THREAD

**Discussion by sFritsch09** [vercel/next.js#49107] [App Router]
File-based metadata is not working on production. I put opengraph-image.jpg, favicon.ico and icon.png in the app dir like described in beta docs: https://beta.nextjs.org/docs/api-reference/metadata#file-based-metadata

My Next App is deployed with kubernetes and I also put the metadataBase URL inside /app/layout.js. In localhost my favicon.ico is successfully resolved and can be accessed with localhost:3000/favicon.ico.

<img width="1804" alt="Screenshot 2023-05-02 at 21 35 48" src="https://user-images.githubusercontent.com/58191603/235768100-f3146b37-bed3-4892-a39f-300a4843bcde.png">

Next version: 13.3.4

**Comment by advancingu** [comment]
Probably a duplicate of https://github.com/vercel/next.js/issues/48962.

**Comment by huozhi** [comment]
Thanks for reporting, let's track it in #48962

**Comment by huozhi** [comment]
13.4.0 seem to resolve the issue, closed this as it's confirmed in the linked issue.
